docs/core/diagnostics/built-in-metrics-system-net.md

```diff
@@ -0,0 +1,137 @@
+---
+title: System.Net Metrics
+description: Review the metrics available for System.Net
+ms.topic: reference
+ms.date: 9/21/2023
+---
+
+# System.Net Metrics
+
+This article describes the networking metrics built-in for <xref:System.Net> produced using the
+<xref:System.Diagnostics.Metrics?displayProperty=nameWithType> API. For a listing of metrics based on the alternate [EventCounters](event-counters.md) API,
+see [here](available-counters.md).
+
+- [Meter: `System.Net.NameResolution`](#meter-systemnetnameresolution) - Metrics for DNS lookups
+  * [Instrument: `dns.lookups.duration`](#instrument-dnslookupsduration)
+- [Meter: `System.Net.Http`](#meter-systemnethttp) - Metrics for outbound networking requests using HttpClient
+  * [Instrument: `http.client.open_connections`](#instrument-httpclientopen_connections)
+  * [Instrument: `http.client.connection.duration`](#instrument-httpclientconnectionduration)
+  * [Instrument: `http.client.request.duration`](#instrument-httpclientrequestduration)
+  * [Instrument: `http.client.request.time_in_queue`](#instrument-httpclientrequesttime_in_queue)
+  * [Instrument: `http.client.active_requests`](#instrument-httpclientactive_requests)
+
+## Meter: `System.Net.NameResolution`
```

Comment:
>## Meter: `System.Net.NameResolution [](http://example.com/codeflow?start=0&length=36)

Do we have any other place having a code sample for how to read these metrics? if not, will be good to track having that and link it to this doc.

Reply:
We have https://learn.microsoft.com/en-us/dotnet/core/diagnostics/metrics-collection.
Its not specifically tailored to reading names from the doc and then collecting those metrics, but its in the ballpark.